src/status_im/contexts/shell/activity_center/notification/common/view.cljs

```diff
@@ -33,35 +35,27 @@
            swipe-button
            swipeable-ref
            style]}]
-  (fn [_ ^js drag-x]
-    (let [{:keys [height] :as extra} (extra-fn)
-          opacity                    (.interpolate drag-x interpolation-opacity)
-          translate-x                (.interpolate drag-x interpolation-translate-x)]
+  (fn [_ drag-x]
+    (let [extra       (extra-fn)
+          opacity     (oops/ocall drag-x :interpolate interpolation-opacity)
```

Comment:
FYI: changing to oops/ocall to be safe for advanced compilation. Note that errors from this macro to instrument args are always disabled in advanced compilation.